kyo-core/shared/src/test/scala/kyo/TagCompactTest.scala

```diff
@@ -25,8 +25,5 @@ class TagCompactTest extends Test:
         "Tag[Emit[Int]]" in {
             assert(Tag[Emit[Int]].raw == "!Bv;!?s;!=q;!T0;!U1;!V2;[=!Y5;!W3;!U1;!V2;]")
         }
-        "Tag[IO]" in {
```

Comment:
Should we also update the Tag compact definition to use this value for something else?

Reply:
I've removed it from the compact dictionary